cases/drp-00-05.tex

```diff
@@ -26,32 +34,149 @@ \subsubsection{Intercase dependencies}
+  \end{enumerate}
+  \item{If processing attempt completes, operator tags attempt as potential for release.  The campaign tags are used to make pre-release QA queries simpler.}
+  \item{If processing attempt fails, restart/resubmit on certain errors} \\
+    For LDM-503-2 test,
+    \begin{enumerate}
+        \item{Low level errors are automatically retried by framework (i.e., db connection fails, file transfer to/from data backbone fails).   For other errors, operator decides whether to just resubmit.   While a course-grained pipeline manual restart capability is available, the operator will simply resubmit the entire pipeline appropriately updating the state in the Oracle database.  Output files from previous attempts can be purged from the Data Backbone if no longer needed for debugging.}
+        \item{If a processing attempt is determined by the operator and production scientist to not be able to complete successfully, the attempt is not tagged and the problems are recorded.  All output files from a single failed processing attempt is kept in the Data Backbone for debugging purposes.} 
+    \end{enumerate}
+\end{enumerate}
+
+\paragraph{Non-science QA}
+\begin{enumerate}
+  \item{When the execution finishes, the success of the execution will be verified by checking the existence of the expected output data.
+  For each of the expected data products types (listed in \S\ref{drp-00-10-items})
+  and each of the expected units (PVIs, coadds, etc), the data product will be
+  verified to be non-empty.}
+    \begin{enumerate}
+        \item{Ask the Data Backbone about the files generated by each tract's execution through the pipeline.}
+        \item{Verify the physical and location information in Data Backbone DB matches the Data Backbone filesystem and vice-versa.} 
+    \end{enumerate}
+  \item{Check that for each data product type that the appropriate metadata was saved for each file as defined in \S\ref{sec:configuration}}
+  \item{Check provenance}
+    \begin{enumerate}
+        \item{Generically check that each file can be linked with the task and processing attempt that created it via the Data Backbone.}
+        \item{Generically check that the inputs for each step were saved to the Oracle database.}
+        \item{Specifically check conditions of DMS-REQ-0106.}
+    \end{enumerate}
+  \item{A brief summary of the execution metrics, such as the number of executions of each executable, execution time usage, wallclock per tract,
+         etc. will be gathered and examined.}
+  \item{Any failure of non-science QA can cause an attempt to be removed from the campaign tag, marked as a bad attempt, and be resubmitted.} 
+\end{enumerate}
+
+\paragraph{First-order Science QA}
+\begin{enumerate}
+  \item{Production Scientist will perform first-order science quality assurance.}
+  \item{These tests were not scheduled as part of LDM-503-2, but will be specified and used in future work.}
```

Comment:
...ah. Maybe this answers my question above. I think if it's not necessary to do this work, we shouldn't record it here. (That doesn't mean it shouldn't be done if it's useful for other purposes, just that we don't need to confuse things by mentioning it.)

Reply:
The goal in putting it here was a placeholder since the LDF plans to provide first-order science QA testing as part of future test runs of pipeline processing.